src/app/components/UserPasswordChange.js

```diff
@@ -53,44 +34,48 @@ class UserPasswordChange extends Component {
-                <FormattedMessage
-                  id="passwordChange.successMsg"
-                  defaultMessage="You're all set. Now you can log in with your new password."
-                />
-              </CardContent>
-              <CardActions className="user-password-change__actions">
-                <Button color="primary" onClick={handleSignIn}>
-                  <FormattedMessage id="passwordChange.signIn" defaultMessage="Got it" />
-                </Button>
-              </CardActions>
-            </Card> :
-            <Card className="user-password-change__card">
-              <CardContent>
-                <img alt="" src={stringHelper('LOGO_URL')} className="user-password-change__logo" />
+      <FormattedMessage id="passwordChange.title" defaultMessage="Change password">
+        { label => (
+          <PageTitle prefix={label}>
+            <StyledPasswordChange>
+              { this.state.showConfirmDialog ?
+                <Card className="user-password-change__confirm-card">
+                  <CardHeader title={<FormattedMessage id="passwordChange.successTitle" defaultMessage="Password updated" />} />
+                  <CardContent>
+                    <FormattedMessage
+                      id="passwordChange.successMsg"
+                      defaultMessage="You're all set. Now you can log in with your new password."
+                    />
+                  </CardContent>
+                  <CardActions className="user-password-change__actions">
+                    <Button color="primary" onClick={handleSignIn}>
+                      <FormattedMessage id="passwordChange.signIn" defaultMessage="Got it" />
+                    </Button>
+                  </CardActions>
+                </Card> :
+                <Card className="user-password-change__card">
+                  <CardContent>
+                    <img alt="" src={stringHelper('LOGO_URL')} className="user-password-change__logo" />
 
-                <span className="user-password-change__title">
-                  <FormattedMessage id="passwordChange.title" defaultMessage="Change password" />
-                </span>
+                    <span className="user-password-change__title">
+                      <FormattedMessage id="passwordChange.title" defaultMessage="Change password" />
+                    </span>
 
-                <ChangePasswordComponent
-                  type="reset-password"
-                  show_current_password={false}
-                  token={token}
-                  show_confirm={this.showConfirm.bind(this)}
-                />
-              </CardContent>
-            </Card>
-          }
-        </StyledPasswordChange>
-      </PageTitle>
+                    <ChangePasswordComponent
+                      type="reset-password"
+                      show_current_password={false}
+                      token={token}
+                      show_confirm={this.showConfirm.bind(this)}
```

Comment:
I know this is not a change in this PR, but why those props use snake case instead of camel case?

Reply:
I noticed it, but I didn't want to touch it in this commit.